templates/minimal/runtime/src/lib.rs

```diff
@@ -26,20 +26,24 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 extern crate alloc;
 
 use alloc::{vec, vec::Vec};
-use frame::{
-	deps::frame_support::{
-		genesis_builder_helper::{build_state, get_preset},
-		runtime,
-		weights::{FixedFee, NoFee},
-	},
-	prelude::*,
-	runtime::{
-		apis::{
-			self, impl_runtime_apis, ApplyExtrinsicResult, CheckInherentsResult,
-			ExtrinsicInclusionMode, OpaqueMetadata,
+use polkadot_sdk::{
+	polkadot_sdk_frame as frame,
+	polkadot_sdk_frame::{
+		deps::frame_support::{
```

Comment:
The whole value of `polkadot-sdk-frame` is in using `prelde::*`, please see https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_frame/index.html